### DISCUSSION TITLE
Listen to events without integration

### DISCUSSION THREAD

**Discussion by deepinder10** [vercel/vercel#6271] [Help]
How to listen to events like deployment complete on my custom webhook. I know it can be done with integrations, but i don't want to create one as it seems from the docs that the integrations will be public and i have to submit that for review.
I just want a simple event handling to publish events to my webhook POST url.

**Comment by trezy** [comment]
Unfortunately, I don't think there's a way to do this without an integration. I've been trying to figure this one out myself and ended up building an integration anyway.

However, you aren't *required* to publish your integration. You can just never submit it for review and only use it on your own projects.

**Comment by cristiammercado** [comment]
I had the same problem, I looked for already created integrations and I couldn't find them either. I had to create my own integration, I share it with you in case anyone would like to use it. It's not published. Link to integration: https://vercel.com/integrations/webhook
